### PR TITLE
Release 1.6.1 snapshot.20201215.5318.0.547abc97

### PR DESCRIPTION
**Pull request by miklos-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: LATEST [hunk 1]
```diff
@@ -1,6 +1,7 @@
 a8ae8e4aa782b8949d32aec3ae05a3211a2306f7 1.8.0-snapshot.20201214.5841.0.a8ae8e4a
 38455e8ca91ca3c351a6d15bd9ec25070386fc6c 1.8.0-snapshot.20201208.5840.0.38455e8c
 e75d42ddc3150c0e054c35cc8c5afcf523ed5702 1.7.0
+547abc97d96ad9ab4ed6e5fb6acf4c05ae85b639 1.6.1-snapshot.20201215.5318.0.547abc97
 aafe46a53625d6c8a31e99d5cf392ca63e822d9f 1.6.1-snapshot.20201021.5317.0.aafe46a5
```
Comment (cocreature) [inline LATEST:5]:
```suggestion
547abc97d96ad9ab4ed6e5fb6acf4c05ae85b639 1.6.1-snapshot.20201215.5318.0.547abc97
```
One line per stable release should be enough.

**Review by cocreature — APPROVED** [review]
Thanks!

File: release/RELEASE.md [hunk 2]
```diff
@@ -285,7 +285,7 @@ patches we backport to the 1.0 release branch).
 
    > Manual tests passed on Windows.
 
-1. If the release is bad, delete the release from [the releases page]. Mention
+1. If the release is bad, delete the release from the [releases page]. Mention
```
Comment (garyverhaegen-da) [inline release/RELEASE.md:288]: Please make this as a separate PR. I'm actually not sure how CI will react to a release PR that is not a single commit. 🙁

Comment (miklos-da) [reply release/RELEASE.md:288]: Well, I had to add the annoying empty changelog markers...

Comment (gerolf-da) [reply release/RELEASE.md:288]: You can amend the previous commit to add the changelog markers. The release commit should only contain the change to the `LATEST` file.

Comment (garyverhaegen-da) [reply release/RELEASE.md:288]: From [the release detection step](https://dev.azure.com/digitalasset/daml/_build/results?buildId=64226&view=logs&j=e74c1ec1-54ae-5943-cedf-e785163662c7&t=68519945-eab9-5f94-7ad3-9048857f8c49):
```
Setting 'is_release' to 'false'
```

Comment (cocreature) [reply release/RELEASE.md:288]: empty commits also work for adding changelog entries

Comment (garyverhaegen-da) [reply release/RELEASE.md:288]: I reread the [code for that step](https://github.com/digital-asset/daml/blob/master/azure-pipelines.yml#L343-L381). You can have as many commits as you want, but the total effect of your PR must be a single line of change to the `LATEST` file.

**Pull request closed without merging**